mint.json

```diff
@@ -125,7 +125,8 @@
         "inbox/react/components",
         "inbox/react/preference",
         "inbox/react/multiple-tabs",
-        "inbox/react/advanced-configuration"
+        "inbox/react/advanced-configuration",
+        "inbox/react/migration-guide"
```

Comment:
I feel that we should move the migration guide to the second page in the sidebar wdyt?

Reply:
🤔 Usually, I find the "migration guide" at the bottom as the migration section becomes obsolete after some time(like after a month or two) once almost all the users are on the new version.